server/controllers/article_controller.js

```diff
@@ -50,18 +50,10 @@ export const addArticleAnnotation = (articleId, annotationId) => {
 // If user is null, return public annotations.
 // Returns a promise.
 
-export const getArticleAnnotations = (user, uri, topLevelOnly) => {
-  const conditions = {};
```

Comment:
I think they were meant to be used to make sure you can only see annotations on an article if you are in the group that those annotations were posted to OR if the annotations are public. We actually might want those conditions.